packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/users-and-attesters.md

```diff
@@ -2,18 +2,16 @@
 
 ## Actors in Unirep
 
-### User 👤
+### Users 👤
+**receive reputation** from attesters and are able to **prove received reputation**.
 
-There are users who **receive reputation** and **prove received reputation**.
+* Users sign up by calling `userSignUp` on the Unirep contract with a signup proof.
+* Users' `identityCommitment` is revealed at this time and recorded in the contract to prevent double signup.
+* The identity commitment does not reveal the actual `identity` of the user but allows the user to prove their identity in the circuit.
 
-* Users sign up by calling `userSignUp` in Unirep contract with a signup proof.
-* User's `identityCommitment` is revealed at this time and it will be recorded in the contract to prevent double signup.
-* The identity commitment will not reveal the actual `identity` of the user but at the same time allow user to prove identity in the circuit.
+### Attesters 👑
+can be thought of as `applications`. Attesters define their own reputation systems and are able to **give attestations** to users, which become the users' reputation.
```

Comment:
> which become the users' reputation.

Which are combined to become the users' reputation.

Reply:
✅